terraform/environment/terraform.tfvars.json

```diff
@@ -24,7 +24,7 @@
       "caseworker_front_certificate_domain_name": "caseworker.preproduction.refunds.opg.service.justice.gov.uk",
       "caseworker_front_dns": "caseworker.preproduction.refunds",
       "aurora_serverless_auto_pause": false,
-      "aurora_serverless_deletion_protection": true,
+      "database_deletion_protection": true,
```

Comment:
The intention is to generate good prod-like data in preprod, not restore prod back, but this will be needed for our migrations. I'm going to have to turn off delete protection on aurora, then on a separate PR remove them.